src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -70,8 +129,54 @@ export default {
   methods: {
     updateArmEnabled: function (enabled) {
       this.arm_enabled = enabled;
+      //ra_command, have all nulls in position. And all zeros in velocity array
+      if (this.arm_enabled === false) {
+        const zeroCommandData = {
+          name: [
+            "joint_a",
+            "joint_b",
+            "joint_c",
+            "joint_d",
+            "joint_e",
+            "joint_f"
+          ],
+          position: [null, null, null, null, null, null],
+          velocity: [0, 0, 0, 0, 0, 0],
+          effort: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
+        };
+        var zeroCommandMsg = new ROSLIB.Message(zeroCommandData);
+        this.jointstate_pub.publish(zeroCommandMsg);
+      }
```

Comment:
This is only called once so I don't have a super strong opinion on this but it may be nice to abstract sending a zero-command into a function. Sending a zero command is a very contained thing that we may want to use later

Reply:
Thinking about it more, it may be best to have a watch function on armEnabled that checks if it's false and then calls the send zero command function. Then this function here (updateArmEnabled) becomes a one-liner that could be put in the html